src/excmds.ts

```diff
@@ -922,7 +922,7 @@ export function composite(...cmds: string[]) {
 
 /** Please use fillcmdline instead */
 //#background
-export function showcmdline() {
```

Comment:
This function appears to be invoked locally only (which seems to be intended, according to the comment).